### DISCUSSION TITLE
Imagine Bundle and AssetMapper

### DISCUSSION THREAD

**Discussion by ebitkov** [liip/LiipImagineBundle#1605] [Q&A] [answered]
The current recommendation to handle assets is to use the AssetMapper component (https://symfony.com/doc/current/frontend/asset_mapper.html), which integrates with the Twig `asset` function and automatically adds versioning to images.

As stated in the documentation, the bundle should automatically detect the versioning and handle it: https://symfony.com/bundles/LiipImagineBundle/current/asset-versioning.html.

But for me it seems not to do it. I get a "404 - Source image for path could not be found" error.

So I'm wondering: Does this bundle support the new AssetMapper component? And if yes, am I the only one, who has problem with it?

**Comment by dbu** [comment]
i have not used this functionality yet. it is possible there are bugs with it - would need somebody to debug this and propose a fix.

**Comment by wouterSkepp** [comment]
This bundle will look for a configuration under the `framework.assets.version` / `framework.assets. json_manifest_path` keys. 

The new AssetMapper component stores it's configuration under `framework.asset_mapper` and is currently not supported by this bundle.

**Comment by ebitkov** [comment, marked as answer]
Issue created: https://github.com/liip/LiipImagineBundle/issues/1608
